INSTALL

```diff
@@ -84,8 +84,8 @@ Set default libraries:
                                                  ltdl, set install path. NEST uses the
                                                  ltdl for dynamic loading of external
                                                  user modules. [default=ON]
-    -Dwith-python=[OFF|ON|2|3]                   Build PyNEST. To set a specific Python
-                                                 version, set 2 or 3. [default=ON]
+    -Dwith-python=[OFF|ON|3]                     Build PyNEST. Options ON and 3 are
```

Comment:
Do we still need the 3? Is it not cleaner with just OFF and ON?

Reply:
"3" option has been removed.